### ISSUE TITLE
Support BIP-0329 wallet label import

### ISSUE THREAD

**Issue by 0xB10C** [sommerfelddev/sentrum#1]
Cool project! A feature I think could be useful would be to have an import for BIP-0329 wallet labels or even use them by default. They allow you to remember why you were tracking a wallet/utxo/xpub and easily allow people to share public BIP-0329 data sets.

One idea that comes to mind is to have a BIP-0329 of (presumably) Satoshis coins and to get an alert if they move. These coins could be a public BIP-0329 list.

https://github.com/bitcoin/bips/blob/master/bip-0329.mediawiki

**Comment by sommerfelddev** [comment]
Good idea but right now sentrum's bitcoin functionality is 100% dependent on [bdk](https://github.com/bitcoindevkit/bdk) so they would need to support it first.

**Comment by sommerfelddev** [comment]
Just for tracking purposes, this is blocked by https://github.com/bitcoindevkit/bdk/issues/1114
